cli/tools/registry/pm.rs

```diff
@@ -22,8 +22,7 @@ use crate::args::CacheSetting;
 use crate::args::Flags;
 use crate::factory::CliFactory;
 use crate::file_fetcher::FileFetcher;
-use crate::lsp::jsr::CliJsrSearchApi;
-use crate::lsp::search::PackageSearchApi;
```

Comment:
I think there was a change to make these modules public in `lsp/mod.rs`. Can we make them private now?